src/AltinnCore/Runtime/RestControllers/InstancesController.cs

```diff
@@ -422,7 +426,7 @@ private async Task DispatchEvent(string eventType, Instance instance)
                 InstanceId = instance.Id,
                 InstanceOwnerId = instance.InstanceOwnerId,
                 UserId = userId,
-                WorkflowStep = instance.Process?.CurrentTask,
+                WorkflowStep = instance.Process?.CurrentTask?.ProcessElementId,
```

Comment:
I'm gonna need an explanation for this notation.. 
does this cover the null check ? The way process is implemented now current task might be null.

Reply:
It (`?.`) is a new operator in C#6 called Null-conditional. It has a built in null check. If `CurrentTask` is null the object traversal is stopped and the sequence returns null.